presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -53,6 +53,7 @@ statement
         (WHERE booleanExpression)?
         (ORDER BY sortItem (',' sortItem)*)?
         (LIMIT limit=INTEGER_VALUE)?                                   #showPartitions
+    | KILL DIGIT_IDENTIFIER                                            #killQuery
```

Comment:
Currently presto QueryId begins with numbers, so I made it DIGIT_IDENTIFIER, maybe I should change it to identifier.